iothub/service/src/Common/Exceptions/DeviceMaximumQueueDepthExceededException.cs

```diff
@@ -59,12 +59,6 @@ internal DeviceMaximumQueueDepthExceededException()
         {
         }
 
-        private DeviceMaximumQueueDepthExceededException(SerializationInfo info, StreamingContext context)
```

Comment:
We're going to redo exceptions as another step, and remove all of these.

If we kept these, I take it these ctors are used for de/serialization, which would explain why they don't have references. I'd suggest leaving them be for now.